source/install/ee-prod-rhel-6.rst

```diff
@@ -317,6 +317,9 @@ Set up NGINX with SSL (Recommended)
 7. Add the following line to cron so the cert will renew every month
   * ``crontab -e``
   * ``@monthly /home/YOURUSERNAME/letsencrypt/letsencrypt-auto certonly --reinstall -d yourdomainname && sudo service nginx reload``
+8. Check that your SSL certificate is set up correctly
+  * Test the SSL certificate by visiting a site such as `https://www.ssllabs.com/ssltest/index.html <https://www.ssllabs.com/ssltest/index.html>`_
+  * If there’s an error about the missing chain or certificate path, there is likely an intermediate certificate missing that needs to be included
```

Comment:
People really struggle with this, propose asking CH for review/verification, I think he mentioned [there was a post in forums](https://forum.mattermost.org/t/android-mattermost-client-version-incompatible/2151/4?u=it33) that had the precise instructions to fix.